### ISSUE TITLE
Feat Request: RUST

### ISSUE THREAD

**Issue by djhunter67** [bootdotdev/curriculum#24]
# RUST:
I am an avid Rusr developer as well as a full `boot.dev` member.  Please add Rust to the curriculum.  I have written several full-stack, command line, and embedded development with Rust.  It seems as if it is a miss to not include a language as modern as Rust.

**Comment by Desk888** [comment]
Yes I will be happy to see some Rust courses too, maybe something like "Introduction to Rust" or something light to begin with or a project-based course like "Build A Load Balancer in Rust" etc.. I think Rust it's a language that is becoming more and more in demand.

**Comment by nuk380y** [comment]
Especially with the Linux Kernel Development Team integrating Rust kmods alongside C, this could be a _huge_ benefit to future Boot.Dev students.

**Comment by Youg-Otricked** [comment]
It would be awsome to have rust on course, but it would be really hard to teach, as rust is 'A little" hard

**Comment by Chinchzilla** [comment]
+1 for rust course

**Comment by IbyA** [comment]
+1 for rust course

**Comment by HarshitRV** [comment]
+1 for rust course

**Comment by Supra-MK3** [comment]
+1 for rust course

**Comment by bamcmanus** [comment]
+1 for rust course

**Comment by ngcammayo** [comment]
+1 for rust course
